### ISSUE TITLE
allow bind of `expanded` attribute on amp-accordion

### ISSUE THREAD

**Issue by YonahKarp** [ampproject/amphtml#20703]
## Describe the new feature or change to an existing feature you'd like to see

`amp-accordion` has an `expanded` attribute that is used to open the accordion. It is added on click of the accordion "header" and can be set manually, to start open. 

Please allow this value to be bound. 

## Additional context

My use case (for me) is an accordion inside of an `amp-list`. The accordions contain a list of filters that, when applied, reload the `amp-list` (filters included, for more relevance). When the data is reloaded, I want to leave the accordions' state as they were before. Allowing bind of `expanded` will make this possible

**Comment by morsssss** [comment]
Just hit this issue myself... +1 to making it bindable!

**Comment by Joelsz** [comment]
Is there a working example of this somewhere?

I tried it and it doesn't seem to expand or to add the `expanded` attribute when the `[data-expand]` evaluates to true. Is this the expected behavior?

**Comment by valse** [comment]
Hi, I have the same issue and I can't expand the accordion inside the list by the mustache template data; I'm setting on the accordion' section:
`[data-expand]="{{isExpanded}}"`
but the accordion still collapsed... any suggestion?

**Comment by caroqliu** [comment]
[Here is an example playground sample.](https://playground.amp.dev/#share=PCFkb2N0eXBlIGh0bWw+CjxodG1sIOKaoT4KPGhlYWQ+CiAgPG1ldGEgY2hhcnNldD0idXRmLTgiPgogIDx0aXRsZT5NeSBBTVAgUGFnZTwvdGl0bGU+CiAgPGxpbmsgcmVsPSJjYW5vbmljYWwiIGhyZWY9InNlbGYuaHRtbCIgLz4KICA8bWV0YSBuYW1lPSJ2aWV3cG9ydCIgY29udGVudD0id2lkdGg9ZGV2aWNlLXdpZHRoLG1pbmltdW0tc2NhbGU9MSxpbml0aWFsLXNjYWxlPTEiPgogIDxzdHlsZSBhbXAtYm9pbGVycGxhdGU+Ym9keXstd2Via2l0LWFuaW1hdGlvbjotYW1wLXN0YXJ0IDhzIHN0ZXBzKDEsZW5kKSAwcyAxIG5vcm1hbCBib3RoOy1tb3otYW5pbWF0aW9uOi1hbXAtc3RhcnQgOHMgc3RlcHMoMSxlbmQpIDBzIDEgbm9ybWFsIGJvdGg7LW1zLWFuaW1hdGlvbjotYW1wLXN0YXJ0IDhzIHN0ZXBzKDEsZW5kKSAwcyAxIG5vcm1hbCBib3RoO2FuaW1hdGlvbjotYW1wLXN0YXJ0IDhzIHN0ZXBzKDEsZW5kKSAwcyAxIG5vcm1hbCBib3RofUAtd2Via2l0LWtleWZyYW1lcyAtYW1wLXN0YXJ0e2Zyb217dmlzaWJpbGl0eTpoaWRkZW59dG97dmlzaWJpbGl0eTp2aXNpYmxlfX1ALW1vei1rZXlmcmFtZXMgLWFtcC1zdGFydHtmcm9te3Zpc2liaWxpdHk6aGlkZGVufXRve3Zpc2liaWxpdHk6dmlzaWJsZX19QC1tcy1rZXlmcmFtZXMgLWFtcC1zdGFydHtmcm9te3Zpc2liaWxpdHk6aGlkZGVufXRve3Zpc2liaWxpdHk6dmlzaWJsZX19QC1vLWtleWZyYW1lcyAtYW1wLXN0YXJ0e2Zyb217dmlzaWJpbGl0eTpoaWRkZW59dG97dmlzaWJpbGl0eTp2aXNpYmxlfX1Aa2V5ZnJhbWVzIC1hbXAtc3RhcnR7ZnJvbXt2aXNpYmlsaXR5OmhpZGRlbn10b3t2aXNpYmlsaXR5OnZpc2libGV9fTwvc3R5bGU+PG5vc2NyaXB0PjxzdHlsZSBhbXAtYm9pbGVycGxhdGU+Ym9keXstd2Via2l0LWFuaW1hdGlvbjpub25lOy1tb3otYW5pbWF0aW9uOm5vbmU7LW1zLWFuaW1hdGlvbjpub25lO2FuaW1hdGlvbjpub25lfTwvc3R5bGU+PC9ub3NjcmlwdD4KICA8c2NyaXB0IGFzeW5jIHNyYz0iaHR0cHM6Ly9jZG4uYW1wcHJvamVjdC5vcmcvdjAuanMiPjwvc2NyaXB0PgogIDxzY3JpcHQgYXN5bmMgY3VzdG9tLWVsZW1lbnQ9ImFtcC1hY2NvcmRpb24iIHNyYz0iaHR0cHM6Ly9jZG4uYW1wcHJvamVjdC5vcmcvdjAvYW1wLWFjY29yZGlvbi0wLjEuanMiPjwvc2NyaXB0PgogIDxzY3JpcHQgYXN5bmMgY3VzdG9tLWVsZW1lbnQ9ImFtcC1iaW5kIiBzcmM9Imh0dHBzOi8vY2RuLmFtcHByb2plY3Qub3JnL3YwL2FtcC1iaW5kLTAuMS5qcyI+PC9zY3JpcHQ+CiAgPHN0eWxlIGFtcC1jdXN0b20+CiAgICBoMSB7CiAgICAgIG1hcmdpbjogMXJlbTsKICAgIH0KICA8L3N0eWxlPgo8L2hlYWQ+Cjxib2R5Pgo8YW1wLWFjY29yZGlvbj4KICA8c2VjdGlvbgogICAgW2RhdGEtZXhwYW5kXT0ic2VjdGlvbk9uZSIKICAgIG9uPSJleHBhbmQ6QU1QLnNldFN0YXRlKHtzZWN0aW9uT25lOiB0cnVlfSk7Y29sbGFwc2U6QU1QLnNldFN0YXRlKHtzZWN0aW9uT25lOiBmYWxzZX0pIgogID4KICAgIDxoMj5TZWN0aW9uIDE8L2gyPgogICAgPHA+QnVuY2ggb2YgYXdlc29tZSBjb250ZW50PC9wPgogIDwvc2VjdGlvbj4KICA8c2VjdGlvbj4KICAgIDxoMj5TZWN0aW9uIDI8L2gyPgogICAgPGRpdj5CdW5jaCBvZiBhd2Vzb21lIGNvbnRlbnQ8L2Rpdj4KICA8L3NlY3Rpb24+CiAgPHNlY3Rpb24+CiAgICA8aDI+U2VjdGlvbiAzPC9oMj4KICAgIDxkaXY+QnVuY2ggb2YgYXdlc29tZSBjb250ZW50PC9kaXY+CiAgPC9zZWN0aW9uPgo8L2FtcC1hY2NvcmRpb24+CjxidXR0b24gb249InRhcDpBTVAuc2V0U3RhdGUoe3NlY3Rpb25PbmU6IHRydWV9KSI+RXhwYW5kIHNlY3Rpb24gMTwvYnV0dG9uPgo8YnV0dG9uIG9uPSJ0YXA6QU1QLnNldFN0YXRlKHtzZWN0aW9uT25lOiBmYWxzZX0pIj5Db2xsYXBzZSBzZWN0aW9uIDE8L2J1dHRvbj4KPC9ib2R5Pgo8L2h0bWw+Cg==) The amp-accordion [documentation page](https://amp.dev/documentation/components/amp-accordion/) is missing the `amp-bind` extension, hence why it does not work there. This is filed already in https://github.com/ampproject/amp.dev/issues/4828.

**Comment by valse** [comment]
Hi, thanks for your help but that is not my issue: I have an `amp-list` with an `amp-accordion` in its mustache template (an accordion list); even if I set the accordion `expanded` attribute in the template, it still collapsed.
